view/js/dropzone-factory.js

```diff
@@ -0,0 +1,63 @@
+var DzFactory = function () {
+	this.createDropzone = function(element, target, maxImagesize) {
```

Comment:
Please rename `element` to `dropSelector` and `target` parameter to `textareaSelector`.
```suggestion
	this.createDropzone = function(dropSelector, textareaSelector, maxImagesize) {
```